docs/hubs-cloud-aws-estimated-cost-charts.md

```diff
@@ -2,6 +2,7 @@
 id: hubs-cloud-aws-estimated-cost-charts
 title: AWS Estimating Costs and Cost Charts (Alpha)
 sidebar_label: Estimating Cost Charts (Alpha)
+description: Explains how to estimate the costs of a Hubs Cloud instance
```

Comment:
This should probably be

Explains how to estimate the costs of a Hubs instance on AWS